app/src/main/res/values/dimens.xml

```diff
@@ -19,5 +19,58 @@
     <dimen name="respreview_margin">16dp</dimen>
     <dimen name="rescount_marginStart">45dp</dimen>
     <dimen name="rescount_marginTop">29dp</dimen>
+    <dimen name="activity_height">0dp</dimen>
+    <dimen name="drinkName_horizontalBias">0.0</dimen>
+    <dimen name="drinkName_verticalBias">0.032</dimen>
+    <dimen name="drinkfavourite_verticalBias">0.695</dimen>
+    <dimen name="shortdescription_verticalBias">0.0</dimen>
+    <dimen name="fragmentprofileImageview_height">100dp</dimen>
+    <dimen name="fragmentprofileImageview_marginStart">16dp</dimen>
+    <dimen name="fragmentprofileImageview_marginTop">16dp</dimen>
+    <dimen name="fragmentprofileImageview_marginEnd">16dp</dimen>
+    <dimen name="fragmentprofileImageview_marginBottom">5dp</dimen>
+    <dimen name="fragmentprofileTextview_height">50dp</dimen>
+    <dimen name="fragmentprofileTextview_marginStart">16dp</dimen>
+    <dimen name="fragmentprofileTextview_marginTop">5dp</dimen>
+    <dimen name="fragmentprofileTextview_marginEnd">16dp</dimen>
+    <dimen name="fragmentprofileTextview_horizontalBias">1.0</dimen>
```

Comment:
As you can see we have many 16dp named differently, so we can keep it as a single dimension?